sdk/keyvault/Azure.Security.KeyVault.Administration/src/BackupOperation.cs

```diff
@@ -116,26 +118,41 @@ public override BackupResult Value
         public override Response GetRawResponse() => _response;
 
         /// <inheritdoc/>
-        public override Response UpdateStatus(CancellationToken cancellationToken = default)
-        {
-            if (!HasCompleted)
-            {
-                Response<FullBackupDetailsInternal> response = _client.GetBackupDetails(Id, cancellationToken);
-                _value = response.Value;
-                _response = response.GetRawResponse();
-            }
-
-            return GetRawResponse();
-        }
+        public override Response UpdateStatus(CancellationToken cancellationToken = default) =>
+            UpdateStatusAsync(false, cancellationToken).EnsureCompleted();
 
         /// <inheritdoc/>
-        public override async ValueTask<Response> UpdateStatusAsync(CancellationToken cancellationToken = default)
+        public override async ValueTask<Response> UpdateStatusAsync(CancellationToken cancellationToken = default) =>
+            await UpdateStatusAsync(true, cancellationToken).ConfigureAwait(false);
+
+        private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToken cancellationToken = default)
         {
             if (!HasCompleted)
             {
-                Response<FullBackupDetailsInternal> response = await _client.GetBackupDetailsAsync(Id, cancellationToken).ConfigureAwait(false);
-                _value = response.Value;
-                _response = response.GetRawResponse();
+                try
+                {
+                    Response<FullBackupDetailsInternal> response = async ?
+                        await _client.GetBackupDetailsAsync(Id, cancellationToken).ConfigureAwait(false)
+                        : _client.GetBackupDetails(Id, cancellationToken);
+
+                    _value = response.Value;
+                    _response = response.GetRawResponse();
+                }
+                catch (RequestFailedException ex)
+                {
+                    _requestFailedException = ex;
+                    throw;
+                }
+                catch (Exception ex)
+                {
+                    _requestFailedException = new RequestFailedException("Unexpected Failure.", ex);
```

Comment:
1. Sentence casing, i.e. not "F" but "f".
2. IIRC, shouldn't end in a period. .NET typically doesn't, but check if we have precedent (probably don't).